### ISSUE TITLE
[Bug]: Create Integration broken on latest Create

### ISSUE THREAD

**Issue by Kanzaji** [XFactHD/FramedBlocks#275]
### Minecraft version

1.19

### Forge version

43.2.8

### FramedBlocks version

6.8.3

### Describe the bug

Not sure how that relates to the gameplay (I'm guessing you can't use FramedBlocks in Create Contraptions / it crashes) however FramedBlocks error out in latest Create (0.5.1b) due to a huge update to Create!
Logs attached below

### How to reproduce the bug

1. Install 1.19.2 Create 0.5.1b and 6.8.3 Framed Blocks on 43.2.8+ Forge
2. Launch the game
3. Check the logs, it will show `An error occured while initializing Create integration!`
```
java.lang.NoClassDefFoundError: com/simibubi/create/content/contraptions/components/structureMovement/MovingInteractionBehaviour
	at xfacthd.framedblocks.common.compat.create.CreateCompat.commonSetup(CreateCompat.java:50) ~[FramedBlocks-6.8.3.jar%23420!/:6.8.3]
	at xfacthd.framedblocks.common.compat.CompatHandler.commonSetup(CompatHandler.java:28) ~[FramedBlocks-6.8.3.jar%23420!/:6.8.3]
	at xfacthd.framedblocks.FramedBlocks.onCommonSetup(FramedBlocks.java:96) ~[FramedBlocks-6.8.3.jar%23420!/:6.8.3]
	at net.minecraftforge.eventbus.EventBus.doCastFilter(EventBus.java:260) ~[eventbus-6.0.3.jar%23106!/:?]
	at net.minecraftforge.eventbus.EventBus.lambda$addListener$11(EventBus.java:252) ~[eventbus-6.0.3.jar%23106!/:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:315) ~[eventbus-6.0.3.jar%23106!/:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:296) ~[eventbus-6.0.3.jar%23106!/:?]
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.acceptEvent(FMLModContainer.java:107) ~[javafmllanguage-1.19.2-43.2.8.jar%23553!/:?]
	at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$10(ModContainer.java:122) ~[fmlcore-1.19.2-43.2.8.jar%23552!/:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) [?:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796) [?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) [?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) [?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) [?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) [?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) [?:?]
Caused by: java.lang.ClassNotFoundException: com.simibubi.create.content.contraptions.components.structureMovement.MovingInteractionBehaviour
	at jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:137) ~[securejarhandler-2.1.4.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:137) ~[securejarhandler-2.1.4.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	... 16 more
```

### Expected behavior

It will not error out 😅

### Crash log

[latest.log](https://github.com/XFactHD/FramedBlocks/files/11585208/latest.log)

**Comment by XFactHD** [comment]
This is a known issue and has been fixed in https://github.com/XFactHD/FramedBlocks/commit/6bd726f1399bba466d315dc821bce9ced2ffa9fc. I had intended to release this as part of the next large update but since that is turning out to be very unpredictable in terms of release date, I'm thinking about releasing this fix earlier in a minor update. I'll leave this open until I have come to a decision on this.

In terms of gameplay the impact of the integration breaking is fairly minor, specifically that framed levers cannot be interacted with on contraptions and that certain things like framed signs may not stick to contraptions correctly.

**Comment by Kanzaji** [comment]
Ah alrighty, good to know its already fixed!
Waiting for a release then 😄

**Comment by XFactHD** [comment]
A new version with the fix for this is now available on CurseForge.

**Comment by funnyman8** [comment]
I'd like to add on that this also affects the 1.18 version, which 5.1 also released for.

**Comment by XFactHD** [comment]
I totally missed that, thanks for the reminder. I'll get an update for that out as well in a few hours

**Comment by XFactHD** [comment]
Fixed version for 1.18.2 is now available on CurseForge. Thanks again for the heads-up
